src-docs/src/views/page/page.js

```diff
@@ -10,13 +10,30 @@ import {
   EuiPageHeader,
   EuiPageSideBar,
   EuiTitle,
+  EuiButton,
 } from '../../../../src/components';
 
 export default () => (
   <EuiPage>
     <EuiPageSideBar>SideBar nav</EuiPageSideBar>
     <EuiPageBody component="div">
-      <EuiPageHeader pageTitle="Page title" />
+      <EuiPageHeader
+        iconType="logoElastic"
+        pageTitle="Page title"
+        tabs={[
+          {
+            label: 'Section one',
+            isSelected: true,
+          },
+          {
+            label: 'Section two',
+          },
+        ]}
```

Comment:
Using tabs in conjunction with `pageTitle` and `rightSideContent` is actually not one I'd like to promote on this first example. So I'm just going to remove the tabs option from this one.
```suggestion
```